app/assets/main.css

```
@@ -86,10 +88,68 @@
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .cta-glint::after {
+    animation: glint 5s linear infinite;
+  }
+}
+
+@keyframes glint {
+  0% {
+    --glint-angle: 236deg;
+    opacity: 1;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)